scikeras/wrappers.py

```diff
@@ -1641,6 +1644,6 @@ def r_squared(y_true, y_pred):
             tf.math.squared_difference(y_true, tf.math.reduce_mean(y_true, axis=0)),
             axis=0,
         )
-        return tf.math.reduce_mean(
+        return 1 - tf.math.reduce_mean(
```

Comment:
Shouldn't tests break with this change? I don't think this function should be called "`r_squared`" if this value is returned.

Reply:
There was a single test that "broke", it's "fixed" in `test_scoring.py`.

This said, and in relation to https://github.com/adriangb/scikeras/pull/189#discussion_r576316582, I've debated if this function should even be included in SciKeras. Do you think it's valuable to be able to use this as a loss function, as opposed to just using MSE? I don't think there is any real advantage, and thus this function should probably be removed. I forget why I added it in the first place; I'm guessing it was out of naivety.

Reply:
I opened #197 to address this. All R^2 changes are reverted from this PR.